app/views/wiki/edit.html.erb

```diff
@@ -60,8 +60,12 @@
             <input aria-label="Wiki Title" tabindex="1" name="title" type="text" class="form-control" id="title" value="<% if @node && @node.latest %><%= @node.latest.title %><% else %><%= params[:title] %><% end %>">
           </div>
 
-          <script>
+          <script type="application/javascript">
             (function(){
+              $E.initialize({
```

Comment:
I decided to initialize $E here to point toward the main comment form on /wiki/new once the page loads.